diff --git a/projects/demo/Banded b/projects/demo/Banded
old mode 100755
new mode 100644
index be738c7..519b21e
--- a/projects/demo/Banded
+++ b/projects/demo/Banded
@@ -1 +1 @@
-wish < tcl/Banded.tcl | demo BandedWG -or -ip
\ No newline at end of file
+wish < tcl/Banded.tcl | ./demo BandedWG -or -ip
diff --git a/projects/eguitar/ElectricGuitar b/projects/eguitar/ElectricGuitar
index dfc6437..9f078b2 100755
--- a/projects/eguitar/ElectricGuitar
+++ b/projects/eguitar/ElectricGuitar
@@ -1 +1 @@
-wish < tcl/EGuitar.tcl | ./eguitar -or -ip
+wish < tcl/Eguitar.tcl | ./eguitar -or -ip
